flexget/plugins/api_tvmaze.py

```diff
@@ -119,6 +203,11 @@ def update(self, series, session):
         self.last_update = datetime.now()
 
         self.genres[:] = get_db_genres(series.genres, session)
+        if series.cast and series.cast.people:
+            self.actors[:], self.characters[:] = get_db_actors_and_characters(series.cast.people, session)
```

Comment:
I think something like this, http://docs.sqlalchemy.org/en/latest/orm/basic_relationships.html#association-object except extra_data will actually be another foreign key column.